backend/strapi-deploy-actions

```diff
@@ -9,7 +9,7 @@ SLACK="/home/ifixit/Code/Exec/slackAnnounce.php"
 git fetch --tags --quiet
 
 SUBJECT=$(git rev-list --format="%s" --max-count=1 "$STRAPI_COMMIT" | tail -1)
-COMMIT_LINK="<https://$BUILD_REPO/commit/$STRAPI_COMMIT|${SUBJECT}>"
+COMMIT_LINK="<https://github.com/iFixit/react-commerce/commit/$STRAPI_COMMIT|${SUBJECT}>"
 DEPLOY_LOG_TS=$($SLACK --message="Strapi Deploy: Deploying ${COMMIT_LINK} - 📃 <$BUILD_LOG_URL|build log>")
```

Comment:
The `BUILD_LOG_URL` is also not set, [ref](https://ifixit.slack.com/archives/C8KBFH1JT/p1704311410290419).

Reply:
Nice catch!